docs/titan-basics/workspace-windows.md

```diff
@@ -117,7 +117,7 @@ On all consoles apart from Sapphire Touch and Titan Go, **context menu
 buttons** appear which contain useful functions for the current
 active window and change depending on the window. 
 
-On Diamond 9 the context menu buttons are in the top left hand corner of the right
+On Diamond 9 and Diamond 7 the context menu buttons are in the top left hand corner of the right
```

Comment:
The context menu on the D7 is not in the same place as on the D9. It is just in each individual window more similar to the sapphire.